spec/commands/add_spec.rb

```diff
@@ -239,4 +239,14 @@
       expect(err).not_to include("You may also need to change the version requirement specified in the Gemfile if it's too restrictive")
     end
   end
+
+  describe "when a gem is added and cache_all is true" do
```

Comment:
I think this fix is independent from the `cache_all` setting?

That setting (when false) prevents`bundle cache` from caching `:git` and `:path` gems. It was added in https://github.com/bundler/bundler/commit/d351e68fa0a5df6de7aff587effce0801297848a, although I see no explanation on _why_.

Anyways, I don't think you need to set it here.

Reply:
You're right! Dropped that and updated the commit messages.